src/mavis/annotate/file_io.py

```diff
@@ -58,6 +57,64 @@ def load_masking_regions(*filepaths: str) -> Dict[str, List[BioInterval]]:
     return regions
 
 
+def load_known_sv(*filepaths: str) -> Dict[str, List["BreakpointPair"]]:
+    """
+    loads a standard MAVIS or BED file input to a ist of known breakpoints.
+
+    Standard BED file requirements:
+    reads a file of regions. The expect input format for the file is tab-delimited and
+    the header should contain the following columns
+
+    - chr: the chromosome
+    - start: start of the region, 1-based inclusive
+    - end: end of the region, 1-based inclusive
+    - name: the name/label of the region
+
+    For example:
+
+    .. code-block:: text
+
+        #chr    start       end         name
+        chr20   25600000    27500000    centromere
+    Args:
+        filepath: path to standard MAVIS format file
+    Returns:
+        a dictionary with {str:{BreakpointPair}}
+    """
+    regions = {}
+    for filepath in filepaths:
+        header = set(pd.read_csv(filepath, nrows=1, sep='\t').columns)
+        mavis_header = {'break1_chromosome', 'break2_chromosome'}
+        bed_header = {'chr', 'start', 'end', 'name'}
+        if mavis_header.issubset(header):
+            bpps = read_bpp_from_input_file(filepath, expand_orient=True, expand_svtype=True)
+            for bpp in bpps:
+                chr_list = [bpp.break1.chr, bpp.break2.chr]
+                regions.setdefault(tuple(chr_list), []).append(bpp)
+
+        elif bed_header.issubset(header):
+            logger.warning('BED file support will be deprecated in future versions')
+            df = pd.read_csv(
+                filepath, sep='\t', dtype={'chr': str, 'start': int, 'end': int, 'name': str}
+            )
+            for col in bed_header:
+                if col not in df:
```

Comment:
maybe a dumb question. Wouldn't pandas error if the columns didn't exist since we added them to the dtype already?

Reply:
No in this implementation. As long as the types match, the file will be read in.